src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -287,11 +273,7 @@ public FormValidation doCheckTargetHost(@QueryParameter("targetHost") final Stri
         return FormValidation.ok("Valid Host");
     }
 
-    private boolean validateTargetPort(String targetPort) {
-        if(!DatadogClient.ClientType.DSD.name().equals(reportWith)) {
-            return true;
-        }
-
+    public static boolean validatePort(String targetPort) {
         return StringUtils.isNotBlank(targetPort) && StringUtils.isNumeric(targetPort);
```

Comment:
We could consider `0` as being invalid as well. That would also help to revert that change: https://github.com/jenkinsci/datadog-plugin/pull/59/files#diff-1885a11f973c18f396e5002cdbf7ae14R75

Reply:
That makes sense!